src/super_gradients/examples/predict/detection_predict_with_labels.py

```diff
@@ -0,0 +1,26 @@
+from super_gradients.common.object_names import Models
+from super_gradients.training import models
+from pathlib import Path
+
+# Note that currently only YoloX, PPYoloE and YOLO-NAS are supported.
```

Comment:
this comment should be in line 5 (before the model instantiation)

Reply:
Which is exactly where it is.

Reply:
fixed